modules/spacelift/README.md

```diff
@@ -25,72 +25,94 @@ components:
-        policies_available: []
-        policies_enabled: []
+        drift_detection_schedule: ["0 4 * * *"]
+        external_execution: true
+        git_repository: infra # TODO: replace with your repository name
+        git_branch: main
+
+        # List of available default Rego policies to create in Spacelift.
+        # These policies are defined in the catalog https://github.com/cloudposse/terraform-spacelift-cloud-infrastructure-automation/tree/master/catalog/policies
+        # These policies will not be attached to Spacelift stacks by default (but will be created in Spacelift, and could be attached to a stack manually).
+        # For specify policies to attach to each Spacelift stack, use `var.policies_enabled`.
+        policies_available:
+          - "git_push.proposed-run"
+          - "git_push.tracked-run"
+          - "plan.default"
+          - "trigger.dependencies"
+          - "trigger.retries"
+
+        # List of default Rego policies to attach to all Spacelift stacks.
+        # These policies are defined in the catalog https://github.com/cloudposse/terraform-spacelift-cloud-infrastructure-automation/tree/master/catalog/policies
+        policies_enabled:
+          - "git_push.proposed-run"
+          - "git_push.tracked-run"
+          - "plan.default"
+          - "trigger.dependencies"
+
+        # List of custom policy names to attach to all Spacelift stacks
+        # These policies must exist in `components/terraform/spacelift/rego-policies`
         policies_by_name_enabled: []
 
+        runner_image: 000000000000.dkr.ecr.us-west-2.amazonaws.com/infra #TODO: replace with your ECR repository
+        spacelift_component_path: components/terraform
+        stack_config_path_template: stacks/%s.yaml
+        stack_destructor_enabled: false
+        worker_pool_name_id_map:
+          <core-region-auto>-spacelift-worker-pool: SOMEWORKERPOOLID #TODO: replace with your worker pool ID
+        infracost_enabled: false # TODO: decide on infracost
+        terraform_version: "1.3.5"
+        terraform_version_map:
+          "1": 1.3.5
+
+        # These could be moved to $PROJECT_ROOT/.spacelift/config.yml
+        before_init:
+          - spacelift-configure
+          - spacelift-write-vars
+          - spacelift-tf-workspace
+        before_plan:
+          - spacelift-configure
+        before_apply:
+          - spacelift-configure
+
     # Manages policies, admin stacks, and core OU accounts
     spacelift:
       metadata:
         component: spacelift
         inherits:
           - spacelift-defaults
+      settings:
+        spacelift:
+          policies_by_id_enabled:
+            # This component also creates this policy so this is omitted prior to the first apply
+            # then added so it's consistent with all admin stacks.
+            - trigger-administrative-policy
       vars:
         enabled: true
-        # Use context_filters to split up admin stack management
```

Comment:
Why was this removed? This is a pretty helpful example and we use it in infra-live.